assets/src/application/ui/input/Button/IconButton/style.scss

```diff
@@ -5,6 +5,14 @@
 
 @import 'application/ui/styles/mixins/breakpoints.scss';
 
+:root {
+	--ee-icon-button-size-tiny: 30px;
+	--ee-icon-button-size-small: 36px;
+	--ee-icon-button-size: 42px;
+	--ee-icon-button-size-big: 48px;
+	--ee-icon-button-size-huge: 54px;
+}
```

Comment:
although this is ok for this case, in the long term it would be great to centralize these variables and derive from a set of global variables, this might be safer and more maintanable;
***
I do like the approach semantic UI had:
https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/elements/button.less#L585
https://github.com/Semantic-Org/Semantic-UI/blob/master/src/themes/default/elements/button.variables#L46
***
bootstrap has also interesting approach, but it's difficult to follow in few cases:
https://github.com/twbs/bootstrap/blob/master/scss/_buttons.scss#L5
https://github.com/twbs/bootstrap/blob/master/scss/_variables.scss#L552

Reply:
ya I waffled about just copying those constants into this file, but wanted to talk to you first to get your input on best ways to handle all of this

Reply:
see https://github.com/eventespresso/event-espresso-core/issues/2961